pageserver/src/tenant/timeline.rs

```diff
@@ -5028,31 +5023,31 @@ impl Timeline {
                 // PITR is not set. Retain the size-based limit, or the default time retention,
                 // whichever requires less data.
                 GcCutoffs {
-                    pitr: std::cmp::max(time_cutoff, cutoff_horizon),
-                    horizon: std::cmp::max(time_cutoff, cutoff_horizon),
+                    time: self.get_last_record_lsn(),
```

Comment:
This is more than a rename

Reply:
That's true (it's a subtle fix to how metrics that report pitr history treat a tenant that has no history).